android-base/src/main/java/io/github/droidkaigi/confsched2020/App.kt

```diff
@@ -7,13 +7,21 @@ import com.google.firebase.firestore.FirebaseFirestore
 import com.google.firebase.firestore.FirebaseFirestoreSettings
 import dagger.android.AndroidInjector
 import dagger.android.DaggerApplication
+import io.github.droidkaigi.confsched2020.announcement.ui.subscribeAnnouncementTopic
 import io.github.droidkaigi.confsched2020.di.AppComponent
 import io.github.droidkaigi.confsched2020.di.AppComponentHolder
 import io.github.droidkaigi.confsched2020.di.createAppComponent
+import io.github.droidkaigi.confsched2020.ext.changedForever
+import io.github.droidkaigi.confsched2020.model.SystemProperty
+import io.github.droidkaigi.confsched2020.system.store.SystemStore
 import timber.log.LogcatTree
 import timber.log.Timber
+import javax.inject.Inject
 
 open class App : DaggerApplication(), AppComponentHolder {
+    @Inject
+    lateinit var systemStore: SystemStore
```

Comment:
DroidKaigi 2020 does not use Flux. So we should not use Store and Actions.
This fix will likely be complicated.
I mistakenly attached the easy tag to this issue. 🙇 
It's my mistake, so I think you can stop this PR.

Reply:
Should I do nothing?

Reply:
There are many things to erase.  Since the architecture is not Flux and it is not possible to change the language from the settings at first, it is necessary to remove Action, SystemStore, Dispatcher, CoroutinePlugin, unused LiveData extension functions and functions that depend on them.  Since the amount of change required is large, I thought about closing this PR once.  But if you just delete it, you may not need to make such a big change.